indico/modules/events/registration/util.py

```diff
@@ -161,6 +161,7 @@ def validate_email(self, field):
 
         field_impl = form_item.field_impl
         setattr(RegistrationFormWTF, form_item.html_field_name, field_impl.create_wtf_field())
+    signals.event.registration_form_wtform_created.send(regform, wtform=RegistrationFormWTF)
```

Comment:
I'd pass `management` and `registration` to the signal as well. Just in case it's useful/needed at some point.

Reply:
What about `form_cls` or `wtform_cls` instead of `wtform` (which could also be an *instance* of a form)?